src/Bitmap_cubical_complex/include/gudhi/Bitmap_cubical_complex_base.h

```diff
@@ -467,6 +481,121 @@ class Bitmap_cubical_complex_base {
 
   Top_dimensional_cells_range top_dimensional_cells_range() { return Top_dimensional_cells_range(this); }
 
+  //****************************************************************************************************************//
+  //****************************************************************************************************************//
+  //****************************************************************************************************************//
+  //****************************************************************************************************************//
+  class Vertices_cells_iterator : std::iterator<std::input_iterator_tag, T> {
```

Comment:
I think `Vertices_iterator` is enough, a vertex is a cell. Or it could be `Complex_vertex_iterator` to follow `Simplex_tree`.

Reply:
Indeed!